block/manager.go

```diff
@@ -935,3 +948,31 @@ func (m *Manager) HeaderCache() *cache.Cache[types.SignedHeader] {
 func (m *Manager) DataCache() *cache.Cache[types.Data] {
 	return m.dataCache
 }
+
+// LoadCache loads the header and data caches from disk.
+func (m *Manager) LoadCache() error {
+	gob.Register(&types.SignedHeader{})
```

Comment:
do we need to use gob here? seems like an extra dependency which could be avoided